### ISSUE TITLE
`this._super()` fails in 'classic' computeds

### ISSUE THREAD

**Issue by dfreeman** [pzuraq/ember-decorators-polyfill#15]
With `ember-decorators-polyfill` installed, calling  `this._super()` in a computed property in a classic class no longer returns the value from the definition on the parent.

```js
const Parent = EmberObject.extend({
  foo: computed(function() {
    return 'ok';
  }),
});

const Child = Parent.extend({
  foo: computed(function() {
    // `value` is `'ok'` in both 3.9 and the 3.10 beta,
    // but `undefined` in 3.9 with the polyfill installed
    let value = this._super();
    return value.toUpperCase();
  }),
});

Child.create().foo;
```

**Comment by Herriau** [comment]
Running into the same issue. This breaks the [ember-cp-validations](https://github.com/offirgolan/ember-cp-validations) add-on (see [addon/validations/factory.js#L115](https://github.com/offirgolan/ember-cp-validations/blob/b3a64cc56d60178990796ae293d0f6cfd60a8877/addon/validations/factory.js#L115))

**Comment by pzuraq** [comment]
This should be fixed in v1.0.5!
